### PR TITLE
MM-61526 Fix scrollbar not clickable due to being covered by toaster

### PR DESCRIPTION
**Pull request by lorumic**
#### Summary

Applied the same [style](https://github.com/mattermost/mattermost/blob/master/webapp/channels/src/components/toast/toast.scss#L6-L9) of the `toast` component to the `toast_wrapper` component, so that toasters have a consistent approach and the scrollbar is not covered by the toaster, staying always interactable.

#### Ticket Link

Fixes https://github.com/mattermost/mattermost/issues/29069
Jira https://mattermost.atlassian.net/browse/MM-61526

#### Release Note

```release-note
Fixed scrollbar not clickable when there is a toaster.
```

### CODE REVIEW

File: webapp/channels/src/components/toast_wrapper/toast__wrapper.scss [hunk 1]
```diff
@@ -2,9 +2,9 @@
     position: absolute;
     z-index: 3;
     display: flex;
-    width: 100%;
+    width: calc(100% - 30px);
```
Comment (hmhealey) [inline webapp/channels/src/components/toast_wrapper/toast__wrapper.scss:5]:
```suggestion
    width: calc(100% - 32px);
```

File: webapp/channels/src/components/toast_wrapper/toast__wrapper.scss [hunk 2]
```diff
@@ -2,9 +2,9 @@
...
     align-items: center;
     justify-content: center;
-    margin-top: 16px;
+    margin: 16px 15px 0;
```
Comment (hmhealey) [inline webapp/channels/src/components/toast_wrapper/toast__wrapper.scss:8]:
```suggestion
    margin: 16px 16px 0;
```

**Review by hmhealey — APPROVED** [review]
Thanks for the PR, @lorumic! I'm just going to change the side margins to match the top one here, but this looks great

**Review by M-ZubairAhmed — APPROVED** [review]

**Review by yasserfaraazkhan — APPROVED** [review]
able to click on scroll when the toast message is upimage

**Pull request merged**